TextToSpeech/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+set(PLUGIN_NAME TextToSpeech)
+set(MODULE_NAME ${NAMESPACE}${PLUGIN_NAME})
+
+find_package(${NAMESPACE}Plugins REQUIRED)
+
+add_subdirectory(test)
+
+add_library(${MODULE_NAME} SHARED
+        TextToSpeech.cpp
+        Module.cpp
+        impl/TTSManager.cpp
+        impl/TTSSpeaker.cpp
+        impl/logger.cpp
+        )
+set_target_properties(${MODULE_NAME} PROPERTIES
+        CXX_STANDARD 11
+        CXX_STANDARD_REQUIRED YES)
+
+list(APPEND CMAKE_MODULE_PATH
+        "${CMAKE_CURRENT_SOURCE_DIR}/cmake/")
+
+find_program(PKG_CONFIG_EXECUTABLE NAMES pkg-config DOC "pkg-config executable")
```

Comment:
Why not create FindGstreamer.cmake or similar and use find_package? note, there is PkgConfig find package already.

Reply:
Modified the code to use FindGstreamer.cmake.